Documentation/admin/upgrade.md

```diff
@@ -15,6 +15,23 @@ Please note that the update payload process may affect any or all components in
 
 To learn more about how this process is executed, read about [Operators][operators].
 
+## Production and pre-production channels
+
+Tectonic clusters can be conifigured to track either a production or pre-production update channel for the desired "minor" version of Kubernetes, like `1.8` or `1.7`.
+
+| Name | Workload | Timeframe |
```

Comment:
not "Timeframe" - maybe "Availability"? or "Update"?

Reply:
sure Availability sounds good